setup.py

```diff
@@ -5,6 +5,7 @@
         name="my_dagster_project",
         packages=find_packages(exclude=["my_dagster_project_tests"]),
         install_requires=[
+            "dagit",
```

Comment:
other examples include `dagit` in `extras_require=`, e.g. https://github.com/dagster-io/dagster/blob/master/examples/assets_dbt_python/setup.py#L18, and then in the readme, we recommend using `pip install -e ".[dev]"` to install deps which then will install dagit locally (see readme [here](https://github.com/dagster-io/dagster/tree/master/examples/assets_dbt_python))

including `dagit` in the default `install_requires` would slow down the prod build, because dagster cloud or other deployment set up won't need to install dagit and dagit is a heavy dependency to install. so we separate dagit to extra installs